third_party/cpgfunctionEP-0.3.2/include/cpgfunction/coordinates.h

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
This file creates coordinates for typical configurations: rectangle, open rectangle, U, and L.